src/nginxconfig/generators/conf/general.conf.js

```diff
@@ -124,6 +124,16 @@ export default (domains, global) => {
         config.brotli_types = gzipTypes;
     }
 
+    // Connection header for WebSocket reverse proxy
+    if (domains.some(d => d.reverseProxy.reverseProxy.computed)) {
+        config['# Connection header for WebSocket reverse proxy'] = '';
+        const map = 'map $http_upgrade $connection_upgrade';
+        config[map] = {
+            default: 'upgrade',
+            '""': 'close',
+        };
+    }
```

Comment:
`map` requires a context of `http`. general.conf.js is included in a `server` block, so it cannot go here. I suggest moving this right to the top of website.conf.js -- so that it is in the `http` context, and is only present for the domain that is using the proxy.

Reply:
oh, my bad, i confused the context.

if I put it on top of website.conf.js wouldn't there be multiple `map`s if there are multiple domains that use the reverse proxy?  
is nginx.conf.js maybe a better place?

Reply:
Sure, it can go in nginx.conf.js